package.json

```diff
@@ -38,8 +43,11 @@
     "bower": "^1.8.4",
     "browserify": "^14.4.0",
     "clang-format": "^1.0.55",
-    "cordova": "^6.5.0",
-    "cordova-custom-config": "^5.0.2",
+    "cordova": "~9.0.0",
```

Comment:
Any reason not to use `^` for these, like we do with all the other dependencies?

Reply:
We have seen breakages in the past due to unintended minor version updates. I believe it's safer to only update the patch.